src/cvl/custom_validation/sonic_sflow.go

```diff
@@ -0,0 +1,31 @@
+package custom_validation
+
+import (
+	"net"
+	util "cvl/internal/util"
+ )
+
+
+//Validate ip address of sflow collector ip 
+func (t *CustomValidation) ValidateProperIp(vc *CustValidationCtxt) CVLErrorInfo {
+	ip := net.ParseIP(vc.YNodeVal)
+	if ip == nil {
+	         util.CVL_LEVEL_LOG(0,"%s","IP IS NIL")
+                 return CVLErrorInfo{
+			            ErrCode: CVL_SYNTAX_INVALID_INPUT_DATA,
+				    TableName: "SFLOW_COLLECTOR",
+				    CVLErrDetails : "IP address is not valid",
+			    }
+	}
+
+	if ip.IsLoopback() || ip.IsUnspecified() || ip.Equal(net.IPv4bcast)|| ip.IsMulticast(){
```

Comment:
Please keep the spacing consistent between all the conditions, and put a space before your opening bracket `{` on this line